go/libkb/api.go

```diff
@@ -316,12 +316,12 @@ func (a *InternalAPIEngine) consumeHeaders(resp *http.Response) error {
 func (a *InternalAPIEngine) fixHeaders(arg APIArg, req *http.Request) {
```

Comment:
Do we also want to strip out `User-Agent` and `X-Keybase-Client` headers? Should we just have a policy of "Tor mode = no headers"?